homeassistant/components/nest/legacy/sensor.py

```diff
@@ -167,6 +173,11 @@ def device_class(self):
         """Return the device class of the sensor."""
         return SENSOR_DEVICE_CLASSES.get(self.variable)
 
+    @property
+    def state_class(self):
+        """Return the state class of the sensor."""
+        return SENSOR_STATE_CLASSES.get(self.variable)
```

Comment:
The new way to set these is to use `SensorEntityDescription`. Perhaps these sensors can be rewritten.

Reply:
Yeah or I was going to just delete this part of the  integration, not sure yet...